tests/inference/test_bootstrap_samples.py

```diff
@@ -1,6 +1,7 @@
 import numpy as np
 import pandas as pd
 import pytest
+from estimagic.config import DEFAULT_SEED
```

Comment:
I think we should completely remove the default seed and use whatever you want in tests.